sdk/core/AzureCore/Source/Pipeline/Policies/ContentDecodePolicy.swift

```diff
@@ -194,8 +194,8 @@ public class ContentDecodePolicy: PipelineStage {
         else { return }
         contentType = contentType.lowercased().trimmingCharacters(in: .whitespacesAndNewlines)
         do {
-            if let deserializedData = try deserialize(from: returnResponse, contentType: contentType) {
-                returnResponse.add(value: deserializedData as AnyObject, forKey: .deserializedData)
+            if let deserializedData = try deserialize(from: returnResponse, contentType: contentType) as? AnyObject {
```

Comment:
Unsure why this changed - does this affect behavior or is it just cosmetic?

Reply:
It was necessary with a variant I was trying, but it is no longer necessary so I removed it.